Recruit-Training/Advanced-Recruit-Training/src/BatteryStateDisplayService.cpp

```diff
@@ -26,29 +27,22 @@ void BatteryStateDisplayService::handleBatteryDataReceived(const BatteryData& ba
 {
     batteryStateOfChargeService_.addData(batteryData);
 
-    double milisec = batteryStateOfChargeService_.timeWhenChargedOrDepleted().msec();
-    QString addZero;
-    if(milisec < 100){
-        addZero = "0";
-    }
+    double millisec = batteryStateOfChargeService_.timeWhenChargedOrDepleted().msec();
 
-    if(milisec < 10){
-        addZero = "00";
-    }
+    QString numberMillisec = QString("%1").arg((int)(millisec), 3, 10, QChar('0'));
```

Comment:
I'm pretty sure `.msec()` already returns a int, so just change the type of `millisec`

Reply:
Alternatively, passing in `batteryStateOfChargeService_.timeWhenChargedOrDepleted().msec();` without using a variable is also good.